doc/api/quic.md

```diff
@@ -71,6 +71,8 @@ added: REPLACEME
   * `maxConnectionsPerHost` {number} The maximum number of inbound connections
     per remote host. Default: `100`.
   * `port` {number} The local port to bind to.
+  * `qlog` {boolean} Whether to emit ['qlog'][] events for incoming sessions.
+    (For outgoing client sessions, set `client.qlog`.) Default: `false`.
```

Comment:
Does this need to be here or can we base it solely on the presence of the `qlog` event handler the way we do with things like the keylog support?

Reply:
@jasnell My understanding is that we need to pass the native qlog callback to nghttp2 when setting up the connection, i.e. when the native handle object is created, and we can’t always know if there’s `'qlog'` event handlers at that point?

Reply:
Unless there is a significant performance hit associated with adding the callback I would always add it and include the logic in the qlog handler to check to see if a listener has been added.

Reply:
@jasnell I’ll try to measure the perf impact, but my personal expectation would be that it’s not trivial.

The bigger issue, though, is that it’s not clear at what point we’d know whether there’s going to be a JS event listener or not, and that we’d have to buffer data up until then – which could be indefinitely, I guess?

Reply:
Let's land this as is then but once it lands I'll play around with it a bit more to see if we can reasonably do without the config option.